README.md

```diff
@@ -341,14 +341,36 @@ or raiding, and a visual pull countdown.
 
 It has more fleshed out support for some jobs but is *strongly* a Work In Progress for others.
 
-* Red Mage: Shows white/black mana, tracks procs for Verstone, Verfire and Impact, and shows the state of the melee combo in progress.
-* Warrior: Shows the beast amount, and tracks the remaining Storm's Eye buff time in gcds.
-* Monk: Shows chakra count, remaining greased lightning time, and tracks monk buffs and debuffs.
+<details>
+<summary>Supported Jobs (Click to expand)</summary>
+
+|Job|Feature|
+|:-:|:-:|
+|<img src="https://xivapi.com/cj/1/paladin.png" width="30px"/><br> Paladin|Shows current Oath amount, and atonement stacks. Also tracks Goring Blade DoT. |
```

Comment:
I think I would prefer not to hotlink from xivapi.  Can you use abbreviations (e.g. PLD) or copy icons locally (docs/images/? somewhere else?)

Reply:
OK, I think put into local is good.